web-server/v0.4/src/components/PageHeader/index.js

```diff
@@ -204,6 +205,19 @@ export default class PageHeader extends PureComponent {
     if (tabActiveKey !== undefined) {
       activeKeyProps.activeKey = tabActiveKey;
     }
+    let titleHeader = '';
+    if (selectedControllers !== undefined) {
+      titleHeader = selectedControllers.map(i => (
+        <Tag color="blue" style={{ fontSize: '105%' }}>
+          {i}
+        </Tag>
+      ));
+      titleHeader = <p>Selected Controllers : {titleHeader} </p>;
+    }
+
+    if (selectedControllers === undefined) {
```

Comment:
What about simply referencing this logic as the `else` part of the `if (selectedControllers !== undefined)` logic mentioned above?